shared/src/data/default-state/personnel-templates.ts

```diff
@@ -39,7 +45,9 @@ const notarztPersonnelTemplate = PersonnelTemplate.create(
         height: 80,
         aspectRatio: 1,
     },
-    CanCaterFor.create(1, 1, 4, 'or')
+    CanCaterFor.create(2, 2, 2, 'and'),
+    15,
+    0
```

Comment:
The `notarzt` (with the current `15, 0`) only prioritizes by range, but if it treats e.g. four yellows first and the next two would be red, they would not be treated, but further away there are two greens, they would be treated, as capacity for green is still there)

Reply:
> The `notarzt` (with the current `15, 0`) only prioritizes by range, but if it treats e.g. four yellows first and the next two would be red, they would not be treated, but further away there are two greens, they would be treated, as capacity for green is still there)

And why wouldn't he treat the two red patients that are a little further away than the four yellow patients? A `notarzt` should be able to prioritize the patients and not ignore someone because he is a meter more away than another patient. I thought the `overrideTreatmentRange` should only be used if a player wants personnel to specifically treat a certain person.

Reply:
> And why wouldn't he treat the two red patients that are a little further away than the four yellow patients? A `notarzt` should be able to prioritize the patients and not ignore someone because he is a meter more away than another patient.

@Dassderdie's proposal sounds reasonable. To my mind, a `notarzt` covers an area, looking over the other personnel kneeing besides the laying individual patients and supporting those treatments that need additional competence, i.e. the doctor.

On the other hand, a pure range-based assignment is a reasonable approximation for any user-defined structuring challenge where green patients are moved "out of order" anyway.

Reply:
I change the notarzt now to treat with the same prioritization as the other personnel, just with a bigger range.

Reply:
sounds good.